src/components/shared/page.tsx

```diff
@@ -269,7 +285,7 @@ function Page({ content, imageSalt, getAssetLocation }: Props): ElementWithResou
                 <meta name="viewport" content="initial-scale=1, maximum-scale=5" />
                 <link rel="stylesheet" href="native://fontSize.css" />
             </head>
-            <body>
+            <body className="ios">
```

Comment:
What do full-page interactives use this class for?

Reply:
this would get updated by the client side code using `userAgent`. But I imagine most interactives have the same styles for `ios` and `android` so this would be an enhancement really to apply those styles straight away.